### PR TITLE
Support setting environment variables. Closes thenativeweb/org#516.

### PR DESCRIPTION
**Pull request by grundhoeferj**

### CODE REVIEW

File: lib/application/getConfiguration.js [hunk 1]
```diff
@@ -55,6 +56,18 @@ const getConfiguration = async function (options) {
...
   }
 
+  Object.keys(configuration.environments).forEach(name => {
+    const currentEnvironment = configuration.environments[name];
+
+    if (!currentEnvironment.environmentVariables) {
+      return;
+    }
+
+    const { environmentVariables } = currentEnvironment;
```
Comment (goloroden) [inline lib/application/getConfiguration.js]: Wouldn't it make sense to move this to the line *before* the `if`?

File: lib/application/transformEnvironmentVariables.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const snakeCase = require('lodash/snakeCase');
+
+const transformEnvironmentVariables =
```
Comment (goloroden) [inline lib/application/transformEnvironmentVariables.js]: There something with the indentation is wrong – or, I'd suggest to move the line with `function` to the line of the assignment.

File: lib/application/getConfiguration.js (hunk 1 above)
Comment (grundhoeferj) [reply lib/application/getConfiguration.js]: Sure! 😊

File: lib/application/transformEnvironmentVariables.js (hunk 2 above)
Comment (grundhoeferj) [reply lib/application/transformEnvironmentVariables.js]: Hups... 🤔 it is fixed now!

**Comment by grundhoeferj** [comment]
I fixed the issues 😊

**Pull request merged**